htdocs/install/mysql/migration/5.0.0-6.0.0.sql

```diff
@@ -27,6 +27,7 @@
 ALTER TABLE llx_extrafields ADD COLUMN langs varchar(24);
 
 ALTER TABLE llx_supplier_proposaldet ADD COLUMN fk_unit integer DEFAULT NULL;
+ALTER TABLE llx_projet_task_time ADD COLUMN status  enum('PLANNED','DRAFT','SUBMITTED','APPROVED','CANCELLED','REJECTED','CHALLENGED','INVOICED','UNDERAPPROVAL') DEFAULT 'DRAFT';
```

Comment:
Field with type "enum" is not allowed by dolibar sql rules :
https://wiki.dolibarr.org/index.php/Language_and_development_rules#Table_and_fields_structures.
You can set this to varchar and add the enum arrays as a constant into class Tasks.

Reply:
Thank you but I think I'll come back with a solution based on int.  

In the page you mentionned there is no tinyint or Time  format so I assumed there are also forbidden, aren't they ?

br

Reply:
tinyint can still be used.
For date, only date and datetime are allowed.